extensions/npm/package.json

```diff
@@ -289,6 +289,12 @@
           "tags": [
             "usesOnlineServices"
           ]
+        },
+        "npm.alwaysPerformLookup": {
```

Comment:
This setting makes sense, but the extension sill needs to have a way to activate based on a setting. This not currently possible and is tracked by https://github.com/microsoft/vscode/issues/55299